client/pages/sections/protocols/index.js

```diff
@@ -9,7 +9,7 @@ import Fieldset from '../../../components/fieldset';
 import Repeater from '../../../components/repeater';
 import Controls from '../../../components/controls';
 
-class Form extends Component {
+class Form extends PureComponent {
```

Comment:
What is this?

Reply:
PureComponent calls shouldComponentUpdate and performs a shallow comparison, only rendering if something changed, Component doesn't run a comparison and renders by default